migrations/sql/V7__Add_file_columns.sql

```diff
@@ -0,0 +1,3 @@
+alter table file add column file_size int;
+alter table file add column last_modified_date varchar;
```

Comment:
This should be a `date` type rather than `varchar`, unless there's some reason to keep it as a string?

Reply:
Will need to create new scalar type to support storing date types in DB. New Trello card added (https://trello.com/c/91k6pZif/142-storage-of-client-side-dates-in-graphql) to backlog and note left in code see below: https://github.com/nationalarchives/tdr-prototype-sangria/pull/10/commits/5bcc46fd27ba83a3c6083d41358e658d8cbb4493